src/sentry/integrations/opsgenie/integration.py

```diff
@@ -216,3 +226,27 @@ def build_integration(self, state: Mapping[str, Any]) -> Mapping[str, Any]:
                 "domain_name": f"{name}.app.opsgenie.com",
             },
         }
+
+    def post_install(
```

Comment:
what is the post install doing here that wasn't present before?

Reply:
Adding "team_table" to the config on install.